### ISSUE TITLE
Add a C# linter to CI

### ISSUE THREAD

**Issue by damithc** [PowerPointLabs/PowerPointLabs#1081]
Now that we have a C# coding standard, may be we can add a C# linter to CI?

**Comment by Skaty** [comment]
@damithc I believe the CI is currently running under @kai33 account. We need to set up the CI such that it is using the group's account.

**Comment by damithc** [comment]
> I believe the CI is currently running under @kai33 account. We need to set up the CI such that it is using the group's account.

Yes. Creating org accounts is possible but not easy for AppVeyor. In the meantime, I don't think it affects this PR? You can set up AppVeyor for your own fork for testing this PR.

**Comment by Skaty** [comment]
Microsoft has a static code analysis tool called StyleCop, but I do not know if the coding standard adopted by the analyser is the same.

**Comment by kai33** [comment]
Hi Syed, I've updated the team permissions in AppVeyor, can you help check if u can modify its settings?

**Comment by kai33** [comment]
@Skaty Pptlabs is using StyleCop, u can check if it supports the target standard or we need other static analyser

**Comment by Skaty** [comment]
@kai33 I am not part of the team, so I can't check. However, I do understand that StyleCop runs during msbuild, so there is no need to change any settings, other than the `Settings.StyleCop` file.
